MODULE.bazel

```diff
@@ -45,6 +45,12 @@ single_version_override(
     version = "8.16.1",
 )
 
+bazel_dep(name = "rules_jvm_external", version = "6.9")
```

Comment:
We'll also need to set this up for legacy `WORKSPACE` builds. That means updating:

- `scala/deps.bzl`
- `scala/latest_deps.bzl`
- Probably every legacy `WORKSPACE` file in the project

Another reason the legacy `WORKSPACE` system must be destroyed, but I'm not ready to drop support just yet.